activation_logger.go

```diff
@@ -76,13 +77,19 @@ func NewActivation(config ActivationLoggerConfig) (Logger, error) {
 
 	l := &activationLogger{
 		underlying: config.Underlying,
-
-		activations: config.Activations,
```

Comment:
Should this be removed?

Reply:
Whoops! No, not at all. Thanks for catching that!